### PR TITLE
[CortexXDR] add comment_update arg to update_incident command

### PR DESCRIPTION
**Pull request by thefrieddan1**
## Description
Adding an optional argument add_comment to update_incident command.

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-3070

### CODE REVIEW

**Comment by bziser** [comment]
@thefrieddan1  Please add in the description of the PR, link to the Jira issue

File: Packs/Core/ReleaseNotes/1_3_26.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Investigation & Response
+- Updated the Docker image to: *demisto/python3:3.10.9.46807*.
+
+- Moved ***update_incident_command*** and ***update-remote-system-command*** from CoreIRAPIModule to CortexXDRIR.
```
Comment (dansterenson) [inline Packs/Core/ReleaseNotes/1_3_26.md:6]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.9.46807*.
- Moved ***update_incident_command*** and ***update-remote-system-command*** from CoreIRApiModule to CortexXDRIR.
```

File: Packs/CortexXDR/ReleaseNotes/4_10_9.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Added optional argument *add_comment* to ***xdr-update-incident***.
```
Comment (dansterenson) [inline Packs/CortexXDR/ReleaseNotes/4_10_9.md:4]:
```suggestion
- Added an optional *add_comment* argument to the ***xdr-update-incident*** command.
```

**Review by dansterenson — APPROVED** [review]
Nice

**Review by DeanArbel — APPROVED** [review]
Good job

**Comment by ali-sawyer** [comment]
Hi team, I have not seen this change show up in the Marketplace yet – is that expected? I thought it usually took less than 24 hours after the merge.

**Pull request merged**